gridscale/config.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 //Arrays can't be constants in Go, but these will be used as constants
-var hardwareProfiles = []string{"default", "legacy", "nested", "cisco_csr", "sophos_utm", "f5_bigip", "q35", "q35_nested"}
+var hardwareProfiles = []string{"default", "legacy", "nested", "cisco_csr", "sophos_utm", "f5_bigip", "q35"}
```

Comment:
I saw that you already have a different PR for removing the q35_nested profile https://github.com/gridscale/terraform-provider-gridscale/pull/172

Reply:
Ah, it was for v2. Anyway, `q35_nested` was put in gridscale's official docs by mistake, so it cannot be used by the users from the beginning. Which implies no breaking change :D

Reply:
Btw, I have to remove it in this PR because the new version of `gsclient-go` doesn't support that hw profile, and compile-time error will happen.